src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -0,0 +1,56 @@
+import { NextFunction, Request, Response } from "express";
+import { Errors } from "config/errors";
+import { Subscription } from "~/src/models/subscription";
+import { GitHubServerApp } from "~/src/models/github-server-app";
+
+export const GithubCreateBranchOptionsGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+
+	const { jiraHost } = res.locals;
+	const { issue_key: key } = req.query;
+	if (!key) {
+		return next(new Error(Errors.MISSING_ISSUE_KEY));
+	}
+	const servers = await getGitHubServers(jiraHost);
+	const url = new URL(`${req.protocol}://${req.get("host")}${req.originalUrl}`);
+
+	// Only GitHub cloud server connected
+	if (servers.hasCloudServer && servers.gheServerInfos.length == 0) {
```

Comment:
/nit
Not sure `hasCloudServer` is a correct term, as `server` should it be used when comes to `cloud`?

Reply:
I think, here we're trying to do comparison b/w cloud and enterprise, so I preferred to use `cloud`  rather than only `server`

Reply:
/nit we could probably omit `servers.hasCloudServer &&` since we default to the cloud?
Then you could simplify the getGitHubServers to drop {...hasCloudServer: ghCloudSubscriptions.length},

Reply:
`hasCloudServer` is also required on UI whether to display Cloud option or not.